### PR TITLE
Don't publish packages on fluves

### PR DESCRIPTION
**Pull request by johanvdw**
No longer required as install will use artifactory directly. Also means self-hosted gha-runner can be removed.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ env:
   RSYNC_HOST: ijzer
 jobs:
   build-linux:
-    runs-on: [linux]
+    runs-on: [ubuntu-latest]
     container: ghcr.io/watem-sedem/ci/watem-sedem-conda
     env:
       PYTHONPATH: ../CNWS-processing-scripts/pycnws/src
```
Comment (Sachagobeyn) [inline .github/workflows/build.yml:18]: env python path -> ../CNWS-processing-scripts/pycnws/src -> can this be renamed to ../erosion-sediment-model/pywatemsedem/src

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ env:
   RSYNC_HOST: ijzer
```
Comment (Sachagobeyn) [inline .github/workflows/build.yml:12]: in this build script, can you rename references of CN-WS to WaTEM/SEDEM (CN is an extension now)

**Review by Sachagobeyn — COMMENTED** [review]
@johanvdw : please check my minor comments. I would rename:
- 'CN-WS processing' to 'sediment-erosion-model' 
- pycnws to pywatemsedem-vlaanderen
- CN-WS to WaTEM/SEDEM

**Comment by johanvdw** [comment]
I would do renames in a separate pull request. This PR removes the dependency on the github actions runner at fluves, which is no longer active.

**Review by daarende — APPROVED** [review]

**Pull request merged**